api/package.json

```diff
@@ -10,8 +10,9 @@
     "seed": "knex seed:run",
     "start": "node main.js",
     "start-dev": "nodemon ./main.js",
+    "jest": "NODE_ENV=test jest",
     "test": "NODE_ENV=test tap -J --cov --coverage-report=lcov --no-browser --reporter=spec --test-arg=--silent '{,!(node_modules)/**/}*.test.js'",
-    "test-endpoints": "tap --reporter=spec 'endpoint-tests/*/**/*.js'",
+    "test-endpoints": "jest --runInBand '.+\\.endpoint\\.js'",
```

Comment:
We need to run the tests in serial (or "in band," as Jest calls it) so tests aren't modifying the database at the same time.  We want the database in a known state for each test, so to get that, we need them running individually.  Makes things a little slower, but not a lot.